draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -75,6 +81,20 @@ public DraggableView(Context context, AttributeSet attrs, int defStyle) {
     initializeAttributes(attrs);
   }
 
+    /**
+     * Return if user can maximize on click on minimized view.
+     */
+    public boolean isEnableClickToExpand() {
+        return mEnableClickToExpand;
+    }
+
+    /**
+     * Allow to maximize when the user click on minimized view.
+     */
+    public void setEnableClickToExpand(boolean mEnableClickToExpand) {
```

Comment:
Remember to rename this method to setClickToExpandEnabled once mEnableClickToExpand be renamed.